### PR TITLE
Add Readme text to reaction-swag-shop repository.

### PR DESCRIPTION
**Pull request by prinzdezibel**

### CODE REVIEW

**Comment by zenweasel** [comment]
@prinzdezibel I think for now we want to put a big note that this is a WIP

**Comment by zenweasel** [comment]
Could we add a couple of screenshots from the designs just to give people an idea of what it **will** look like when you are done?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+---
+
+
+This repository contains the reaction-swag-shop plugin that's intended as a learning resource for everybody
+who's facing the task of customizing Reaction for her/his own shop implementation. Additionally, this repository drives
+our real swag shop that's offering merchandising products at [swag.getreaction.io](swag.getreaction.io).
+
+
+### Getting started
+If you like to use this shop plugin as a starting point for your own project, please follow the steps outlined below:
```
Comment (zenweasel) [inline README.md]: "If you would like"

Comment (prinzdezibel) [reply README.md]: done

**Pull request merged**